lib/msf/ui/console/command_dispatcher/common.rb

```diff
@@ -142,6 +142,21 @@ def show_options(mod) # :nodoc:
     #print("\nTarget: #{mod.target.name}\n\n")
   end
 
+  # This is for the "use" and "set" commands
+  def module_index(list, index, &block)
```

Comment:
Sure. This was a late refactor, so it's still evolving. It used to be named `module_index_to_name`, but I moved the module-specific functionality back into `modules.rb` and `core.rb`.

I can rename again to suit that better, since I don't see any module-specific code in here anymore. It is meant to be used for modules, but it can take any non-negative-indexed list now.

Reply:
I didn't commit that iteration, so you won't find it in my numerous force pushes, lol. Was all in the working tree.

Reply:
I like `index_from_list`.